### DISCUSSION TITLE
added toggle button on headerAction

### DISCUSSION THREAD

**Discussion by Nadiahsohaini** [filamentphp/filament#12336] [Help]
### Package

Actions

### Package Version

v3

### How can we help you?

![Screenshot 2024-04-15 174221](https://github.com/filamentphp/filament/assets/94894791/e36cb29f-ae5a-4a61-8320-634c6d88a732)

how to do added toggle on headerAction section

**Comment by leandrocfe** [comment]
`ToggleButton` is a form field, not action. 
You can use `ActionGroup` https://filamentphp.com/docs/3.x/actions/grouping-actions#overview

**Reply by Nadiahsohaini** [reply]
![Screenshot 2024-04-16 095941](https://github.com/filamentphp/filament/assets/94894791/25b60d4a-0be9-4c00-84ac-1f1bd0ccb81f)

i want replace the edit to toggle button switch status in edit form header section. Possible to do?

**Comment by dharen008** [comment]
Same question. i hope there's a function like that

**Comment by enrico-b-248** [comment]
I’m running into the same need.
Having a toggle in the header would be really useful, especially when it represents the overall state of the record/resource (e.g. active vs. archived). Keeping it in the form works for now, but I think exposing it in the header would noticeably improve the UX 👍

**Comment by dharen008** [comment]
This might help everyone

```
Section::make( 'SeciontTitle' )
->afterHeader( [
Toggle::make( 'columnName' ),
] )
```

**Reply by enrico-b-248** [reply]
That did the trick, I've completely missed it. Thank you 💯
